manager/controllers/systems_test.go

```diff
@@ -37,6 +38,10 @@ func TestSystemsDefault(t *testing.T) {
 	assert.Equal(t, "8", output.Data[0].Attributes.OSMajor)
 	assert.Equal(t, "1", output.Data[0].Attributes.OSMinor)
 	assert.Equal(t, "8.1", output.Data[0].Attributes.RhsmVersion)
+	assert.Equal(t, "null", output.Data[0].Attributes.StaleTimestamp)
```

Comment:
It would be better to make test so it will return some not-empty data. You can update test_data.sql if needed.

Reply:
I have found out that these info exist in inventory.hosts table. Do you think that it is better to select from that table? as a single source of truth?